administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -3,18 +3,30 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
+COM_JOOMLAUPDATE_AUTOUPDATE_REGISTER_SUCCESS="Successful registered to autoupdate service"
```

We have removed all uses of the redundant word "successful"
```suggestion
COM_JOOMLAUPDATE_AUTOUPDATE_REGISTER_SUCCESS="Registered to autoupdate service"
```